stable/traefik/README.md

```diff
@@ -88,7 +88,8 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `image`                         | Traefik image name                                                   | `traefik`                                 |
 | `imageTag`                      | The version of the official Traefik image to use                     | `1.3.1`                                  |
 | `serviceType`                   | A valid Kubernetes service type                                      | `LoadBalancer`                            |
-| `loadBalancerIP`                | An available static IP you have reserved on your cloud platform      | None                                      |
+| `loadBalancer.IP`               | An available static IP you have reserved on your cloud platform      | None                                      |
```

Comment:
This is an unnecessary breaking change. It should be `loadBalancerIP` and `loadBalancerSourceRanges`. IMHO, this additional level of nesting is just too much. Also, values should start with a lower case letter.

Please see chart examples for reference:
https://github.com/kubernetes/helm/blob/master/docs/examples/nginx/templates/service.yaml#L22-L28
https://github.com/kubernetes/helm/blob/master/docs/examples/nginx/values.yaml#L24-L25